core/isolation/wordpress-adapter-interface.php

```diff
@@ -20,4 +20,12 @@ public function get_option( $option_key );
 	public function add_option( $option_key, $option_value );
 
 	public function update_option( $option_key, $option_value );
+
+	public function get_referer();
+
+	public function add_query_arg( $args, $url );
+
+	public function has_custom_logo();
+
+	public function page_exists( $params );
```

Comment:
Does this belong in the WP Adpater? There is no WP function called `page_exists`

Reply:
moved page_exists, but i keep has_custom-logo here